<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTCH-5CD105HTG9" name="restassured.automation.testcases.Restassured_Automation_FinancialStatements" tests="11" failures="1" timestamp="9 Jun 2021 08:12:52 GMT" time="43.501" errors="0">
  <testcase name="FinancialStatements_PatchFinancialStatementsForTheRevision_Status400" time="5.723" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_MoveFinancialStatementsLineItems_Status404" time="3.998" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_PatchTheFinancialStatementsLineItems_Status409" time="4.456" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_PatchTheFinancialStatementsLineItems_Status200" time="4.344" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_PatchTheFinancialStatementsLineItems_Status404" time="4.203" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_LoadTheFinancialStatementsForTheStore_Status200" time="2.971" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_createAnNewFinancialStatementsForTheStore_Status409" time="4.288" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_PatchTheFinancialStatementsLineItems_Status400" time="4.256" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_createAnNewFinancialStatementsForTheStore_Status400" time="2.547" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
  <testcase name="FinancialStatements_createAnNewFinancialStatementsForTheStore_Status200" time="2.624" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements">
    <failure type="java.lang.AssertionError" message="expected [200] but found [409]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [409]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_FinancialStatements.FinancialStatements_createAnNewFinancialStatementsForTheStore_Status200(Restassured_Automation_FinancialStatements.java:107)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- FinancialStatements_createAnNewFinancialStatementsForTheStore_Status200 -->
  <testcase name="FinancialStatements_GetFinancialStatementsForTheRevision_Status200" time="4.091" classname="restassured.automation.testcases.Restassured_Automation_FinancialStatements"/>
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_FinancialStatements -->
